db_stress_tool/db_stress_gflags.cc

```diff
@@ -414,10 +414,10 @@ static const bool FLAGS_kill_random_test_dummy __attribute__((__unused__)) =
     RegisterFlagValidator(&FLAGS_kill_random_test, &ValidateInt32Positive);
 extern int rocksdb_kill_odds;
 
-DEFINE_string(kill_prefix_blacklist, "",
+DEFINE_string(kill_exclude_prefixes, "",
```

Comment:
Unfortunately, this one is effectively in the public API as it is an option to the tool.  I do not know what the policy is on preserving command line options or if this requires an update to the public API documentation (HISTORY.md).

This option shows up via db_stress -help currently...

Reply:
db_stress is a development tool, not a customer tool, so subject to change. Help is autogenerated.